standards/certification/digisov-and-cert.md

```diff
@@ -0,0 +1,120 @@
+standards is also possible, allowing for hybrid cloud setups without technical barriers.
+
+Level 3 drives the work on a comprehensive openly developed open source software stack,
+allowing operators to use, study, change and redistribute the software according to the
+[Four Freedoms](https://en.wikipedia.org/wiki/The_Free_Software_Definition) of free software. We are requiring
+a complete stack that uses [OSI](https://opensource.org/)-approved open source licenses
+as to ensure that users have the four freedoms, the right to use, study, modify, (re)distribute
+the software that drives the cloud stack. To ensure that this does not require extensive
+and expensive forking, we further require the [Four Opens](https://openinfra.dev/four-opens/)
+of the Open Infra Foundation here. The software can be used to provide cloud services
+for others (public cloud) or just for your own community (community cloud) or
+internal (private cloud) needs.
+
+Level 4 addresses the skills and transparency aspects. Operating highly dynamic distributed
+systems in a reliable manner requires knowledge and experience — engineers with these skills
+are scarce. To address this, the SCS team networks operations staff from providers and helps
+to share and distill common knowledge that help everyone to be more successful. SCS has
+thus been driving the [Open Operations](https://openoperations.org) initiative.
+
+Levels 2 and 3 are sometimes related to the term "technological sovereignty",
+indicating the ability to control and shape the technology.
+
+## The SCS certification levels
+
+Corresponding to the levels of digital sovereignty in the SCS taxonomy, SCS defines
+SCS certification levels
+
+1. (Defined outside of the SCS scope)
+2. *SCS-compatible*
+3. *SCS-open*
+4. *SCS-sovereign*
+
+### Why no SCS certification for GDPR?
+
+SCS significantly lowers the bar to offer real cloud services. These can be used internally
+(private cloud) or to offer services for your community, your region or country. The vision
+is to have a network of providers. We expect most if not all of them to be operated in ways
+that fulfill the European GDPR regulation; it is also possible to operate clouds that fulfill
+special regulation, e.g. in the banking or insurance sector.
+
+SCS is not in a position to judge this and thus defines no own label / certificate to
+vouch for regulatory compliance. We typically refer to the
+[ENISA](https://www.enisa.europa.eu/) for GDPR considerations
+and also recommend to take the [Gaia-X](https://gaia-x.eu/) labels into account here.
+
+## Status of SCS certification for cloud operators
+
+As of September 2024, the requirements for *SCS-open* and *SCS-sovereign*
+certification have not been formalized yet.
+
+The technical compatibility validation corresponding to the *SCS-compatible* certification does
+exist since more than a year. There are certificates for two layers of the SCS architecture
```

Comment:
```suggestion
The technical compatibility validation corresponding to the *SCS-compatible* certification has
existed for more than a year. There are certificates for two layers of the SCS architecture
```

better yet (insert correct date)

```suggestion
The technical compatibility validation corresponding to the *SCS-compatible* certification has
existed since 2023. There are certificates for two layers of the SCS architecture
```